internal/pkg/test/tool/require.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package tool
```

Comment:
Sorry to make the same comment as the other code review...

Can you please add documentation for this package stating what it provides (https://rakyll.org/style-packages/ has some good advise on how to document packages, near the bottom of the article; https://github.com/golang/go/wiki/CodeReviewComments#package-comments has more of that, too).

What I would like to avoid is a split where some things are added to "test" (one directory above this) and others are added to "tool" (this).

Reply:
I will add a `doc.go` in other PR

Reply:
> I will add a `doc.go` in other PR

There's no need for an additional PR, please add the documentation here as well.

Reply:
As for the name of the package...

Maybe naming the package `require` helps readability, as that would lead to code that reads like `require.CgroupMount` for example, and it establishes that the contents of the package are assert-type functions.